src/Containers/BoxComposition2/BoxComposition2.tsx

```diff
@@ -0,0 +1,115 @@
+import React,{useState,useEffect} from 'react'
+import styled from 'styled-components'
+
+interface IBoxProps{
+    imgSrc:string;
+    top:number;
+    left:number;
+    width:number;
+    height:number;
+    borderRadius:number;
+}
+
+export interface IBoxComposition2Props{
+    box1:IBoxProps;
+    box2:IBoxProps;
+    box3:IBoxProps;
+}
+
+export const BoxComposition2:React.FC<IBoxComposition2Props>=({box1,box2,box3}):React.ReactElement=>{
+    const [showBox2,setShowBox2]=useState(false)
+    const [showBox3,setShowBox3]=useState(false)
+    useEffect(()=>{
+        setTimeout(()=>{
+            setShowBox2(true)
+        },1000)
+        setTimeout(()=>{
+            setShowBox3(true)
+        },2000)
+    },[])
+    return     (
+        <div>
+            <Box1 {...box1} />
+            {showBox2&&<Box2 {...box2} />}
+            {showBox3&&<Box3 {...box3} />}
+        </div>
+    )
+}
+
+const Box3:React.FC<IBoxProps>=({imgSrc,top,left,width,height,borderRadius}):React.ReactElement=>
+    (
+        <AnimatedDiv3 top={top} left={left}>
```

Comment:
BAD VARIABLE NAMES 
AnimatedDiv3 is not good!!

Reply:
ThirdAnimatedDiv will be ok?

Reply:
No. The name has no MEANING.
What is this div?
Is this the CenterDiv
TopLeftDiv etc?
You should be able to read the code and instantly understand what relates to what